app/views/overview/_service-instance-row.html

```diff
@@ -32,7 +32,7 @@ <h3>
           </div>
         </div>
       </div>
-      <div class="list-pf-details" ng-if="!row.expanded">
+      <div class="list-pf-details" ng-if="!row.expanded && !row.integrations | size">
```

Comment:
Why do we need to check size?

Reply:
We don't want to show bindings and integrations on the same object. So only show bindings if there are no integrations.